File: Leetcode_set1/leetcode_26.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        unique_list=[]
        for i in nums:
          if i not in unique_list:
            unique_list.append(i)
            #print(i)
        nums[:] = unique_list
        return len(nums)

File: Leetcode_set1/leetcode_26.c
int removeDuplicates(int* nums, int numsSize) {
    int arr[numsSize];
    int i=0,j=0;
    for(i=0;i<numsSize;i++)
    {
        if(nums[i]!=nums[j])
        {
            j+=1;
            nums[j]=nums[i];
            
        }
    }
    return j+1;
}